task_2

```diff
@@ -0,0 +1,13 @@
+def task_2():
+    a = input("Input numbers:\n").split()
```

Comment:
Numbers list should be provided as an argument. And also the check if it's an iterable object required.